Sources/LanguageServerProtocolJSONRPC/JSONRPCConnection.swift

```diff
@@ -60,10 +63,16 @@ public final class JSONRPCConnection {
 
   var closeHandler: (() -> Void)! = nil
 
+#if os(Windows)
+  public typealias PipeHandle = HANDLE
+#else
+  public typealias PipeHandle = Int32
+#endif
+
   public init(
     protocol messageRegistry: MessageRegistry,
-    inFD: Int32,
-    outFD: Int32,
+    inFD: PipeHandle,
```

Comment:
Could we use `FileHandle` here? I think with that change, most users of `JSONRPCConnection` could work without `#if os(Windows)`, and we would only need special cases (1) in the implementation here, and (2) possibly in main.swift - I'm not sure if you can get a FileHandle for `realStdout` in a portable way.

Reply:
Sure, I can convert to passing a `FileHandle` instead of a FD, that should make the ergonomics of this API way better IMO.